data/presets/presets/leisure/sports_centre.json

```diff
@@ -4,7 +4,6 @@
         "name",
         "sport",
         "address",
-        "building_area",
```

Comment:
I think rather than removing it completely, it would be best to leave this as "building" so that users can choose whether they want it to be a building or not (but not have it _default_ to being a building).